src/components/Wizzy.js

```diff
@@ -0,0 +1,88 @@
+import React, { Component } from 'react';
+import RichTextEditor from 'react-rte';
+import AceEditor from 'react-ace';
+import 'brace/mode/html';
+import 'brace/theme/solarized_light';
+
+class Wizzy extends Component {
+	constructor(props) {
+		super(props);
+
+		this.state = {
+			wizzyValue: RichTextEditor.createEmptyValue(),
+			html: '',
+			isWizzy: true,
+		};
+
+		this.onRichTextEditorChange = this.onRichTextEditorChange.bind(this);
+		this.renderSwitchButton = this.renderSwitchButton.bind(this);
+	}
+
+	onRichTextEditorChange(value) {
+		if (typeof value === 'string') {
+			this.setState({ 
+				wizzyValue: RichTextEditor.createValueFromString(value, 'html'),
+				html: value, 
+			});
+		} else {
+			this.setState({ 
+				wizzyValue: value,
+				html: value.toString('html'),
+			});
+		}
+
+		if (this.props.onChange) {
+			this.props.onChange(this.state.html);
+		}
+	}
+
+	renderSwitchButton() {
+		if (this.state.isWizzy) {
+			return (
+				<div>
+					<button className="wizzy-switch btn btn-default" onClick={() => this.setState({ isWizzy: !this.state.isWizzy })}>Switch to HTML Editor</button>
```

Comment:
should be something like 

```javascript
onClick={handleWizzyButtonClick}
```

not set on this name btw

Reply:
Done!